Ragnarok/Bragi/Bragi.h

```diff
@@ -4,6 +4,7 @@
 	#include <iostream>
 	#include <errno.h>
 	#include <vector>
+	#include <stack>
```

Comment:
Why do you need the stack here?

Reply:
I was think about something but did not work and i forgot to delete this line

Reply:
oh ok!